workers/tests/unittests/test_config.py

```diff
@@ -31,15 +51,34 @@ def test_get_config_path_set_env(self):
             os.path.join(tempfile.gettempdir(), "config.yaml"),
         )
 
+    @mock.patch.dict("os.environ", {}, clear=True)
+    def test_config_default(self):
+        cfg = config.load_config_default()
+        self.assertIsInstance(cfg, dict)
+        self.assertListEqual(self.keys, list(cfg.keys()))
```

Comment:
This comment applies to future occurrences as well:
Is the order of the keys relevant? `assertListEqual` checks the lists, which are ordered.

Reply:
Order is not necessary. Replaced with just `assertEqual`.

Reply:
Which doesn't change my question, since:

> These methods are used by default when comparing lists or tuples with [assertEqual()](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual).

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertListEqual

Reply:
Now both lists to be compared are sorted beforehand.